pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -106,7 +118,16 @@ class PagerState(
             )
         }
 
-    internal var currentLayoutPageSize by mutableStateOf(0)
+    internal inline val currentLayoutPageSize: Int
+        get() = currentLayoutPageInfo.layoutSize
+
+    internal inline val currentLayoutPage: Int
```

Comment:
I don't think "layout" is adding a lot in this vals – why not just `currentPage`?